### DISCUSSION TITLE
Where I can find description of ALL solara compnents?

### DISCUSSION THREAD

**Discussion by rururu** [widgetti/solara#631] [Q&A]
Dear solara users and team?
Where I can find description of, for example, solara.Text component?
It really works, but I could not find its description in documentation. Usually this information is under API heading (I am from Java ecosystem). But here under API heading is something that far away form ordinary understanding of this term. I like Solara very much. I think that for now it is best GUI framework for python. But its documentation is far from best.
Sincerely,
  Ru

**Comment by iisakkirotko** [comment]
All the components that have documentation pages should be available [here](https://solara.dev/documentation/components). Unfortunately not all components have documentation pages, like `solara.Text`, as you found. For these, you would have to go through the source code under `solara/components`, but there may be missing docstrings there as well.

Improving the documentation is on our todo-list, so hopefully we can make it more complete in the (near) future!

**Reply by rururu** [reply]
Thank you for encouraging information!

**Comment by maartenbreddels** [comment]
Let’s turn this into an issue and document it. Text is a component that pre dates the documentation, and we never got to it. Do you want to open it @rururu ?
